ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+from os import path
+from getpass import getuser
```

Comment:
Avoid `from x import y` if there's not a good reason for it. Here, getuser is used only once, so `getpass.getuser` is definitely not a problem.